firefox-ios/Client/Frontend/Browser/MainMenu/Redux/MainMenuAction.swift

```diff
@@ -35,6 +35,8 @@ enum MainMenuActionType: ActionType {
     case tapShowDetailsView
     case tapToggleUserAgent
     case updateCurrentTabInfo
+    case closeMenu
```

Comment:
These have been removed/renamed in a previous PR. You'll have to remove them and fix with the appropriate updated names (`tapCloseMenu`and `tapNavigateToDestination`)